source/mapping-rules/mapping-rule-options/mapping-rule-filters.txt

```diff
@@ -0,0 +1,132 @@
+.. _rm-mapping-filters:
+
+====================
+Mapping Rule Filters
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In a migration project, you can use mapping rule javascript expressions to filter mapping rules based on document values. Mapping rule filters are available to all mapping rule types.
+
+Mapping rule options allow you to:
+
+- Filter mapping rules based on logical conditions.
+- Limit the number of documents written to a table.
+- Access complex and nested document structure for filtering.
```

Comment:
Is this a leftover from a previous edit? It's describing features unrelated to mapping rule filters.  We should give examples of why you might want to use filters.

Reply:
Hey Tom, I added this suggestion, I think complex nested documents may be wrong based on our previous conversation with David. But cant you both limit and apply logical filters with mapping rules such as Field name = "1"?

Reply:
The first point is correct, although the header "mapping rule options" is confusing. 
The second point is incorrect. We do not write to tables, and the mapping rules may or may not relate to new documents.